auth/session.go

```diff
@@ -82,11 +91,19 @@ func (auth *Authenticator) CreateSession(username string, ttl time.Duration) (*L
 		return nil, err
 	}
 
+	user, err := auth.GetUser(username)
+	if user != nil && user.Disabled() {
+		return nil, base.HTTPErrorf(400, "User is disabled")
+	} else if err != nil {
+		return nil, err
```

Comment:
Do we need to prevent creation of sessions for non-existing users, or could we consider empty as the initial state of session UUID?  (presumably that's also needed for backward compatibility for existing users)

Reply:
I am not sure I understand this question. I added a case to test where the SessionUUID is empty to make backward compatible with existing users.

Reply:
Previously we didn't require the user to exist in order to create a session.  I don't think that needs to change, if we treat "empty" as the initial session UUID value.

Reply:
Decided to not worry about this since the session API prevents you from creating a session before a user. I did a follow up commit so that we don't read the user data twice when creating a session as an enhancement.